app/main.go

```diff
@@ -92,6 +92,9 @@ var (
 	boltdbPath = kingpin.Flag("boltdb-path", "path to BoltDB store, e.g. /data/bolt.db").Envar("BOLTDB_PATH").String()
 
 	ver = kingpin.Flag("version", "print the current version revision").Bool()
+
+	monitoringBackends = kingpin.Flag(
```

Comment:
It's confusing what metrics we're talking about so I would advise changing the flag to `--stats-metric-exporters`